crates/rover-client/src/blocking/client.rs

```diff
@@ -37,7 +36,7 @@ impl GraphQLClient {
     ///
     /// Takes one argument, `variables`. Returns an optional response.
     /// Automatically retries requests.
-    pub fn post<Q>(
+    pub async fn post<Q>(
```

Comment:
Is the `blocking` module containing `async` functions part of the "this is WIP" disclaimer in the README?

Reply:
yep! we're going to have more clean up after the next rc, which will probably make it into the actual next release